vita3k/gui/src/controls_dialog.cpp

```diff
@@ -204,6 +206,8 @@ void draw_controls_dialog(GuiState &gui, EmuEnvState &emuenv) {
     }
 
     ImGui::End();
+
+    emuenv.ctrl.drop_inputs = gui.controls_menu.controls_dialog;
```

Comment:
is same for this, this line no have sense
if this function it set obligatory to true...
you need change this

Reply:
What the fuck? this is the line that SETS drop_inputs to true when the menu is open. Did you even read it? Remove the line and you can very clearly see that if you press up and down while the keyboard menu is open, you can still navigate the UI in the background.

Reply:
you have bad understund me here, i said only you need CHANGE this way, here have no sense
this line just set true all time, you check value of controller dialog bool but it is obligatory on true here
i perfectly more understund you who code works
you code here have no sense, is totaly not logical

Reply:
No, the line gets set to false when the window closes. see my other comment. ImGui::Begin() makes sure of this.

Reply:
i thinks you no understund how works this here
controll dialog is set to true by click on button on menubar.
and it waiting you close dialog it for can false

no idea why you speak of ImGui::Begin() totaly no relate it 
you no need set this here

best way is move this drop input outside both 
with check in other part, like inside handle event if one of both is true set drop to true, is very simple